z88dk-git/lure.sh

```diff
@@ -1,33 +1,36 @@
 name='z88dk-git'
-version='21023.a09ee3d7a'
-release=2
+version='21835.e89d1b4477'
+release=3
 desc='The development kit for Z80 computers'
 homepage='https://github.com/z88dk/z88dk'
 maintainer='Atirut Wattanamongkol <atirut.wattanamnogkol@gmail.com>'
 architectures=('amd64')
 license=('MIT')
 
-sources=('git+https://github.com/z88dk/z88dk?~recursive=true')
+sources=('git+https://github.com/z88dk/z88dk?~recursive=true&~rev=e89d1b4477687747b500fbe0e25fc52fe35056a7')
 checksums=('SKIP')
 
-
-build_deps_ubuntu=('build-essential' 'bison' 'flex' 'libxml2-dev' 'subversion' 'zlib1g-dev' 'm4' 'ragel' 're2c' 'dos2unix' 'texinfo' 'texi2html' 'gdb' 'curl' 'perl' 'cpanminus' 'ccache' 'libboost-all-dev' 'libmodern-perl-perl' 'libyaml-perl' 'liblocal-lib-perl' 'libcapture-tiny-perl' 'libpath-tiny-perl' 'libtext-table-perl' 'libdata-hexdump-perl' 'libregexp-common-perl' 'libclone-perl' 'libfile-slurp-perl' 'make')
+build_deps_ubuntu=('build-essential' 'bison' 'flex' 'libxml2-dev' 'subversion' 'zlib1g-dev' 'm4' 'ragel' 're2c' 'dos2unix' 'texinfo' 'texi2html' 'gdb' 'curl' 'perl' 'cpanminus' 'ccache' 'libboost-all-dev' 'libmodern-perl-perl' 'libyaml-perl' 'liblocal-lib-perl' 'libcapture-tiny-perl' 'libpath-tiny-perl' 'libtext-table-perl' 'libdata-hexdump-perl' 'libregexp-common-perl' 'libclone-perl' 'libfile-slurp-perl' 'pkg-config')
 
 version() {
 	cd "$srcdir/z88dk"
 	git-version
 }
 
 build() {
+	cd "$srcdir/z88dk"
+
 	export BUILD_SDCC=1
 	export BUILD_SDCC_HTTP=1
-	cd "$srcdir/z88dk"
+	cpanm --local-lib=~/perl5 App::Prove CPU::Z80::Assembler Data::Dump Data::HexDump File::Path List::Uniq Modern::Perl Object::Tiny::RW Regexp::Common Test::Harness Text::Diff Text::Table YAML::Tiny
+	eval $(perl -I ~/perl5/lib/perl5/ -Mlocal::lib)
 	./build.sh
 }
 
 package() {
-	eval "$(perl -I$HOME/perl5/lib/perl5 -Mlocal::lib)"
-	cpanm App::Prove File::Path CPU::Z80::Assembler Object::Tiny::RW List::Uniq YAML::Tiny
 	cd "$srcdir/z88dk"
+
+	cpanm --local-lib=~/perl5 App::Prove CPU::Z80::Assembler Data::Dump Data::HexDump File::Path List::Uniq Modern::Perl Object::Tiny::RW Regexp::Common Test::Harness Text::Diff Text::Table YAML::Tiny
+	eval $(perl -I ~/perl5/lib/perl5/ -Mlocal::lib)
 	PREFIX="$pkgdir/usr/local" make install
```

Comment:
Since this is a package and not a manual install, use `/usr` instead of `/usr/local`

```suggestion
	PREFIX="$pkgdir/usr" make install
```

Reply:
The compiler seems to hardcode its config in that directory, so nothing can be done as of now. I'll investigate and see if this is a problem with LURe or Z88DK.

Reply:
Based on the issue you created in the z88dk repo, you should be able to fix this by putting `PREFIX="/usr" DESTDIR="$pkgdir"` before both `./build.sh` and `make install`. I tried it and I was able to successfully compile a C file with `zcc`.

Reply:
I completely forgot to complete this PR. Sorry for the wait, your suggestions worked like a charm.